libs/go/zmscli/role.go

```diff
@@ -66,10 +67,24 @@ func (cli Zms) ShowRole(dn string, rn string, auditLog, expand bool, pending boo
 	if err != nil {
 		return nil, err
 	}
-	var buf bytes.Buffer
-	buf.WriteString("role:\n")
-	cli.dumpRole(&buf, *role, auditLog, indentLevel1Dash, indentLevel1DashLvl)
-	return cli.switchOverFormats(role, buf.String())
+
+	oldYamlConverter := func(res interface{}) (*string, error) {
```

Comment:
Right now I've only implemented the "pass as function" logic in the ShowRole method. After this change is approved I'll implement it in the rest of the functions.